package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "sa-map-editor",
+  "version": "0.1.0",
+  "description": "A tool to generate Twilight Imperium Maps for Shatterd Ascension",
+  "private": true,
+  "directories": {
+    "lib": "lib"
+  },
```

Comment:
What does this do? I can't find the description of this property on https://nodejs.dev/learn/the-package-json-guide

Reply:
It is actually not needed, I think it was in the template I copied from. It's just some metadata. And I think it is wrong since it should point to the src of this lib, not the libs we use.

So I think we should either remove it, or point it to src.

I would sugest just removing it

Thats the description of the lib folder:

> Tell people where the bulk of your library is. Nothing special is done with the lib folder in any way, but it's useful meta info.

Other entrys might be
- bin
- doc
- exsample
- man
- test